src/commonUi/commonStyle/_variables.scss

```diff
@@ -0,0 +1,18 @@
+// font size
```

Comment:
Will this be used for all components or just the button?

Reply:
nah, it's a general styling file. You just need to import it.

Reply:
Okk, perfect! Thanks for clarifying Jeff!